records_mover/records/hints/types.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional, Union, Mapping, Dict
+from typing_extensions import Literal, TypedDict
+from records_mover.mover_types import JsonValue
+
+
+RecordsValue = Optional[Union[bool, str]]
+RecordsHints = Mapping[str, JsonValue]
+MutableRecordsHints = Dict[str, JsonValue]
+
+HintEncoding = Literal["UTF8", "UTF16", "UTF16LE", "UTF16BE",
+                       "UTF16BOM", "UTF8BOM", "LATIN1", "CP1252"]
+
+HintQuoting = Literal["all", "minimal", "nonnumeric", None]
+
+HintEscape = Literal["\\", None]
+
+# TODO: combine this and cli thingie
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker